src/lib/components/SettingsModal.svelte

```diff
@@ -53,12 +53,19 @@
 					You can change this setting at any time, it applies to all your conversations.
 				</p>
 				<p class="text-gray-800">
-					Read more about this model's authors,
+					Read more about model authors,
 					<a
 						href="https://open-assistant.io/"
 						target="_blank"
 						rel="noreferrer"
 						class="underline decoration-gray-300 hover:decoration-gray-700">Open Assistant</a
+					>
+					or
+					<a
+						href="https://ai.meta.com/llama/"
+						target="_blank"
+						rel="noreferrer"
+						class="underline decoration-gray-300 hover:decoration-gray-700">Meta AI</a
```

Comment:
Feels like we could handle this automatically from the `.env` see: https://github.com/huggingface/chat-ui/pull/345/

But not a blocker imo, we can fix this later.